analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ControlCharacterInString.CSharp11.cs

```diff
@@ -15,8 +15,12 @@ void Utf8StringLiterals()
         {
             ReadOnlySpan<byte> Utf8Compliant = "test"u8;
             ReadOnlySpan<byte> Utf8Compliant2 = """test"""u8;
-            ReadOnlySpan<byte> Utf8NonCompliant = "test"u8; // FN
-            ReadOnlySpan<byte> Uft8NonCompliant2 = """test"""u8; // FN
+            ReadOnlySpan<byte> Utf8NonCompliant = "test"u8; // Noncompliant
+            ReadOnlySpan<byte> Uft8NonCompliant2 = """test"""u8; // Noncompliant
+
+            var x = "hello";
+
```

Comment:
Maybe we could get rid of those empty lines to have a cleaner test

Reply:
you are right, I removed them.